test/support/file_endpoint.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Membrane.RTC.Engine.Support.FileEndpoint do
+  @moduledoc false
+
+  # Endpoint that publishes data from a file in realtime. It will start publishing data when it receives message :start.
+  # Buffers from file must have timestamps, because it will be used in Realtimer.
```

Comment:
```suggestion
  # Endpoint that publishes data from a file in the realtime. It will start publishing data when it receives message :start.
  # Buffers from the file must have timestamps, because they will be used by Realtimer.